### PR TITLE
Hot restart operations

### PR DESCRIPTION
**Pull request by erosb**
Related MC issue: https://github.com/hazelcast/management-center/issues/2224

Related IMDG PR: https://github.com/hazelcast/hazelcast/pull/16241

### CODE REVIEW

File: protocol-definitions/MC.yaml [hunk 1]
```diff
@@ -741,3 +741,53 @@ methods:
           since: 2.0
           doc: |
             List of events.
+  - id: 25
+    name: triggerPartialRestart
```
[the remark this answers is not in the document]
Comment (erosb) [reply protocol-definitions/MC.yaml]: Done

File: protocol-definitions/MC.yaml [hunk 2]
```diff
@@ -741,3 +741,53 @@ methods:
...
+    response:
+      params:
+        - name: result
+          type: boolean
+          since: 2.0
+          nullable: false
+          doc: |
+            True if the partial restart succeeded, false otherwise
+  - id: 26
+    name: forceStart
```
[the remark this answers is not in the document]
Comment (erosb) [reply protocol-definitions/MC.yaml]: Done

File: protocol-definitions/MC.yaml [hunk 3]
```diff
@@ -741,3 +741,53 @@ methods:
...
+      retryable: false
+      partitionIdentifier: -1
+    response:
+      params:
+        - name: result
+          type: boolean
+          since: 2.0
+          nullable: false
+          doc: |
+            True if the partial restart succeeded, false otherwise
```
[the remark this answers is not in the document]
Comment (erosb) [reply protocol-definitions/MC.yaml]: Done (I used "successfully initiated")

File: protocol-definitions/MC.yaml [hunk 4]
```diff
@@ -741,3 +741,53 @@ methods:
...
+      retryable: false
+      partitionIdentifier: -1
+    response:
+      params:
+        - name: result
+          type: boolean
+          since: 2.0
+          nullable: false
+          doc: |
+            True if the forced start succeeded, false otherwise
```
[the remark this answers is not in the document]
Comment (erosb) [reply protocol-definitions/MC.yaml]: Done (I used "successfully initiated")

**Comment by puzpuzpuz** [comment]
Closing this one in favor of #279

**Pull request closed without merging**